ext/storage_sources/azure_store/azure_storage_source.cpp

```diff
@@ -354,9 +412,11 @@ azure_remove(WT_FILE_SYSTEM *file_system, WT_SESSION *session, const char *name,
     WT_UNUSED(name);
     WT_UNUSED(flags);
 
-    return 0;
+    std::cerr << "azure_remove: Object: " << name << ": remove of file not supported." << std::endl;
+    return ENOTSUP;
 }
 
+// POSIX rename, not supported for cloud objects.
```

Comment:
Shouldn't we just leave the function pointer in the struct as nullptr? I didn't think this one was mandatory. This possibly goes for remove as well.

Reply:
I might be wrong but is WiredTiger expecting this function to actually be a function?

Reply:
@jiechenbo is `remove` and `rename` mandatory